hello_world.py

```diff
@@ -6,3 +6,8 @@ def hello_world():
 hello_world()
 
 
+#I'm hoping to see comments about this line only
+
+def stupid_program():
+ print 'dummy'
```

Comment:
Reporter: FLAKE8
Rule: 0311
Severity: WARN
File: hello_world.py L12
 

Bad indentation. Found 1 spaces, expected 4

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>